src/scripts/main.js

```diff
@@ -1,3 +1,67 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const addRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const addColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+
+function control() {
+  let rLength = table.rows.length;
```

Comment:
1. there might be a `const` instead of `let`
2. don't use short forms ~rLength~ => `rowLength`
3. these variables are redundant (read another comment)